notion-s3-python/requirements.txt

```
@@ -0,0 +1 @@
+turbine-py==1.7.0
```

```suggestion
turbine-py==1.7.1
```


This bug fix addresses a response parsing bug that only happens from time to time.

---

thanks!